Code/Editor/EditorPreferencesPageViewportManipulator.cpp

```diff
@@ -36,7 +48,55 @@ void CEditorPreferencesPage_ViewportManipulator::Reflect(AZ::SerializeContext& s
                 AZ::Edit::UIHandlers::SpinBox, &Manipulators::m_manipulatorCircleBoundWidth, "Circle Bound Width",
                 "Manipulator Circle Bound Width")
             ->Attribute(AZ::Edit::Attributes::Min, 0.001f)
-            ->Attribute(AZ::Edit::Attributes::Max, 2.0f);
+            ->Attribute(AZ::Edit::Attributes::Max, 2.0f)
```

Comment:
Depending on the min and max values it can be nice to set Attributes::Step as well so the values increment a reasonable amount relative to their range.

Reply:
Ah nice idea, I will get this in as-is but can add step-size as a follow-up. I have tried to pick reasonable min/max values for each but having a step size is a really nice addition too. I might do that with the reset change @amzn-leenguy suggested too :+1: